src/Exceptions/AuthenticationConnectException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Pinnacle\OpenIdConnect\Exceptions;
+
+use Pinnacle\OpenIdConnect\Exceptions\Constants\AuthenticationRequestErrorCode;
+
+class AuthenticationConnectException extends OpenIdConnectException
```

Comment:
I think this was an error in a name change refactor. I renamed it to AuthorizationCodeCallbackException since it's dealing with the AuthorizationCodeCallbackData class.